integration/testcmd/gotest/uinit/gotest.go

```diff
@@ -56,7 +92,13 @@ func runTest() error {
 			return
 		}
 
-		cmd := exec.CommandContext(ctx, path, "-test.v", "-test.timeout", individualTestTimeout.String())
+		args := []string{"-test.v"}
+		coverFile := filepath.Join(filepath.Dir(path), "coverage.txt")
```

Comment:
If I move this into the if-statement I need to duplicate it and put it in the if-statement in line 138 as well because otherwise coverFile will be undefined there.

Reply:
You're right!